.travis.yml

```diff
@@ -73,6 +73,7 @@ before_install:
 before_script:
   - docker exec -t devenv /bin/sh -c "cd /opt/shogun/build; cmake -DCMAKE_INSTALL_PREFIX=$HOME/shogun-build -DENABLE_TESTING=ON $CMAKE_OPTIONS .."
 script:
+  - docker exec -t devenv /bin/sh -c "cd /opt/shogun/scripts; bash ./check_format.sh"
```

Comment:
Actually, I'm not really sure this line will work (since the `check_format.sh` script needs bash).